wordpress-mobile/WordPress-iOS
# [Stats Refresh] Blog stats: show new Post Stats

Fixes #11863

To test:
- Go to Blog Posts.
- On a blog, select `More`.
- Select `Stats`. 
- Verify the new Post Stats is shown.
- Verify tapping the post title opens a webview for the post.


![blog_post_stats](https://user-images.githubusercontent.com/1816888/58996134-29ffbe00-87b4-11e9-833c-d449bfcf19fa.png)


Update release notes:
